slides/en/net/merger/html-to-html/_index.md

```diff
@@ -0,0 +1,97 @@
+{{< blocks/products/pf/feature-page-wrap >}}
+
+{{< blocks/products/pf/feature-page-header h1="Merge HTML in C#" h2="Powerful cross-platform .NET API for merging HTML files using C# code on NET Framework, .NET Core, Windows Azure, Mono or Xamarin Platforms" >}}
+
+{{% blocks/products/pf/feature-page-section h2="Merge HTML to HTML using Aspose.Slides" %}}
+
+[**Aspose.Slides for .NET**](https://products.aspose.com/slides/net/) is a powerful .NET library used to create, convert, merge, and manipulate presentations, HTML files, and other documents. When you merge HTML to HTML, you are effectively combining the contents from 2 documents to obtain one HTML file. Aspose.Slides allows you merge HTML docs in different ways. You get to merge HTML with all their texts, formatting,styles, etc.
+
+{{% /blocks/products/pf/feature-page-section %}}
+
+
+
+
+{{% blocks/products/pf/feature-page-section  h2="Merge HTML to HTML in C#" %}}
+Using [**Aspose.Slides for .NET**](https://products.aspose.com/slides/net/), you can merge HTML files quickly with just a few lines of code
+
+{{% blocks/products/pf/agp/code-block title="C# code for merging HTML to HTML" offSpacer="true" %}}
+```cs
+using (Presentation pres = new Presentation())
+            {
+                pres.Slides.RemoveAt(0); // remove default empty slide
+
+                pres.Slides.AddFromHtml("page_1.html");
+                pres.Slides.AddFromHtml("page_2.html");
+
+                pres.Save("MergedFile.html", SaveFormat.Html);
+            }
+```
+{{% /blocks/products/pf/agp/code-block %}}
+
+{{% /blocks/products/pf/feature-page-section %}}
+
+
+
+
+{{< blocks/products/pf/feature-page-section  h2="How to merge HTML in C#" >}}
+
+
+{{< blocks/products/pf/agp/steps-block-autogen name="" >}}
+
+
+{{< blocks/products/pf/agp/step-autogen >}}
+Install **Aspose.Slides for .NET**. See [**Installation**](https://docs.aspose.com/slides/net/installation/).
+{{< /blocks/products/pf/agp/step-autogen >}}
+
+{{< blocks/products/pf/agp/step-autogen >}}
+Add the library as a reference in your project.
+{{< /blocks/products/pf/agp/step-autogen >}}
+
+{{< blocks/products/pf/agp/step-autogen >}}
+Create an instance of the Presentation class.
+{{< /blocks/products/pf/agp/step-autogen >}}
+
+{{< blocks/products/pf/agp/step-autogen >}}
+Load the HTML files you want to merge.
+{{< /blocks/products/pf/agp/step-autogen >}}
+
+{{< blocks/products/pf/agp/step-autogen >}}
+Save the resulting HTML doc.
+{{< /blocks/products/pf/agp/step-autogen >}}
+
+
+{{< /blocks/products/pf/agp/steps-block-autogen >}}
+
+
+{{< /blocks/products/pf/feature-page-section >}}
+
+
+
+
+{{< blocks/products/pf/agp/other-supported-section title="Merge other files" subTitle="You can also combine files in other formats to get a single file" >}}
+
+{{< blocks/products/pf/agp/other-supported-section-item href="https://products.aspose.com/slides/net/merge/jpg-to-jpg/" name="JPG TO JPG" >}}  
```

Comment:
Please fix the url it should be .../merger/... instead of .../merge/... (for all new pages)